README.md

```diff
@@ -83,39 +83,60 @@ minder auth login
 
 Upon completion, you should see that the Minder Server is set to `api.stacklok.com`.
 
-## Enroll a repository provider
+## Quickstart
 
-Minder supports GitHub as a provider to enroll repositories. To enroll your provider, run:
+Minder provides a "happy path" that guides you through the process of creating your first profile in Minder. 
+In just a few seconds, you will register your repositories and enable secret scanning protection for all of them.
+To do so, run:
 
 ```bash
-minder provider enroll --provider github
+minder quickstart --provider github
```

Comment:
Given that we're doing the enrollment ourselves. Maybe the quickstart command shouldn't require the `--provider` flag, and instead it should just use `github` by default.